pkg/clusterapi/aws/actuator_test.go

```diff
@@ -48,19 +48,21 @@ func init() {
 }
 
 const (
-	testNamespace             = "test-namespace"
-	testClusterDeploymentName = "test-cluster-deployment"
-	testClusterDeploymentUUID = types.UID("test-cluster-deployment-uuid")
-	testClusterID             = "test-cluster-id"
-	testClusterVerName        = "v3-10"
-	testClusterVerNS          = "cluster-operator"
-	testClusterVerUID         = types.UID("test-cluster-version")
-	testRegion                = "us-east-1"
-	testImage                 = "testAMI"
-	testVPCID                 = "testVPCID"
-	testSubnetID              = "testSubnetID"
-	testAZ                    = "us-east-1c"
-	testMachineName           = "testmachine"
+	testNamespace              = "test-namespace"
+	testClusterDeploymentName  = "test-cluster-deployment"
+	testClusterDeploymentUUID  = types.UID("test-cluster-deployment-uuid")
+	testClusterID              = "test-cluster-id"
+	testClusterVerName         = "v3-10"
+	testClusterVerNS           = "cluster-operator"
+	testClusterVerUID          = types.UID("test-cluster-version")
+	testRegion                 = "us-east-1"
+	testImage                  = "testAMI"
+	testVPCID                  = "testVPCID"
+	testSubnetID               = "testSubnetID"
+	testAZ                     = "us-east-1c"
+	testMachineName            = "testmachine"
+	testShutdownBehaviorNodes  = "terminate"
+	testShutdownBehaviorMaster = "stop"
```

Comment:
Because you're in the same go package you can just reference the constants defined in actuator.go with the same values.

Reply:
@dgoodwin Yeah, I thought about that. I didn't do that for 2 reasons:

1) I wanted to ensure that if those constants change to invalid values in the future (e.g. "stopped" instead of "stop"), that the tests fail.

2) I wanted to follow the established patterns already in that test file (e.g. see how "master" is statically defined as the expected string instead of referencing the hostTypeMaster constant [here](https://github.com/twiest/cluster-operator/blob/f7de5f05d383932e980b8664667ce395c267054a/pkg/clusterapi/aws/actuator_test.go#L272)).


That being said, I will absolutely change it to use the other constants if you feel strongly about it.

Reply:
Nah that sounds fine, thanks.